python/tskit/formats.py

```diff
@@ -225,6 +224,17 @@ def _load_legacy_hdf5_v3(root, remove_duplicate_positions):
     return tables.tree_sequence()
 
 
+def get_h5py():
+    try:
+        import h5py
+    except ImportError:
+        raise tskit.FileFormatError(
```

Comment:
I think this is still best as an ImportError - we just raise one with a better error message. There's nothing actually wrong with the file format, we're just missing an import.

Reply:
Heh, I originally had this as ImportError so happy to change back!